Inspection/src/main/res/layout/activity_inspection.xml

```diff
@@ -427,11 +427,63 @@
             android:orientation="horizontal">
 
             <TextView
+                android:id="@+id/textView15"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:gravity="center_horizontal"
+                android:text="THIS SOFTWARE HAS BEEN MODIFIED"
+                android:textAppearance="?android:attr/textAppearanceMedium" />
+        </LinearLayout>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="bottom"
+            android:layout_marginBottom="2dp"
+            android:layout_marginTop="2dp"
+            android:gravity="center_horizontal"
+            android:orientation="horizontal">
+
+            <TextView
+                android:id="@+id/textView14"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_weight=".5"
+                android:gravity="center_horizontal"
+                android:text="OpenFTC Version:"
+                android:textAppearance="?android:attr/textAppearanceMedium" />
+
+            <TextView
+                android:id="@+id/txtOpenFTCVersion"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:gravity="center_horizontal"
+                android:textAppearance="?android:attr/textAppearanceMedium" />
+
+            <FrameLayout
+                android:id="@+id/OpenFTCVersion"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"/>
```

Comment:
What's this FrameLayout for?

Reply:
Whoops; I think it's just leftover from while I was writing this. I don't have a computer with Android Studio handy, but I think it can be removed.

Reply:
Yeah, that's about what I figured it was. Thanks.

Reply:
Ugh, didn't mean to do a review